Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2706 @@
+            result.
+    """
+    context_data: list = []
+    payload = create_payload_for_query(args, 'event')
+    offset = int(payload.pop('offset', ''))
+    limit = int(payload.get('limit', ''))
+
+    raw_response = client.get_event_for_incident(payload)
+    title = compile_command_title_string('Event for Incident', args, int(raw_response.get('total', 0)))
+
+    if page_result := get_data_of_current_page(raw_response.get('result', []), offset, limit):
+        readable_output, context_data = incident_event_readable_output(page_result, title)
+    else:
+        readable_output = 'No Event for Incidents data to present.'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.IncidentEvent',
+        outputs_key_field='event_uuid',
+        outputs=context_data,
+        raw_response=raw_response,
+        ignore_auto_extract=True
+    )
+
+
+def get_incident_list_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Get Incident List
+    Args:
+        client: Symantec EDR on-premise client objectd to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+            result.
+    """
+    context_data: list = []
+    payload = create_payload_for_query(args, 'incident')
+    offset = int(payload.pop('offset', ''))
+    limit = int(payload.get('limit', ''))
+
+    raw_response = client.get_incident(payload)
+    title = compile_command_title_string('Incident', args, int(raw_response.get('total', 0)))
+
+    if page_result := get_data_of_current_page(raw_response.get('result', []), offset, limit):
+        readable_output, context_data = incident_readable_output(page_result, title)
+    else:
+        readable_output = 'No Incidents data to present.'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.Incident',
+        outputs_key_field='apt_incident_id',
+        outputs=context_data,
+        raw_response=raw_response
+    )
+
+
+def get_incident_comments_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Get all comments based on Incident ID
+    Args:
+        client: Symantec EDR on-premise client objectd to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+            result.
+    """
+    context_data: list = []
+    # Get UUID based on incident_id
+    uuid = get_incident_uuid(client, args)
+    incident_id = args.pop("incident_id", None)
+    if uuid is None:
+        raise ValueError("Error: No Incident UUID found. Provide valid Incident IDs.")
```

Comment:
is it a UUID or an ID?
Both terms are used here.  Be consistent.

Reply:
Based on Incident ID's, we need to get UUID for any incident which is Unique Id in order to pass it to the endpoint request

Reply:
is there any case where `get_incident_uuid` returns `None`, rather than raises an exception?

Reply:
Yes, It will also return `None` in case incident_id could not find. I have removed the exception from get_incident_uuid method and it will only handle the value exception from calling method.